magento/magento2
# Add collapsible nav in customer account theme blank

### Description (*)

This additionals aims to bring collapsible nav in mobile view for blank theme magento
This improvement accessibility for user when go around links in mobile (with collapsible nav)
Desktop still remain same UI like before
Move collapsible nav from luma theme to blank theme

### Related Pull Requests


### Fixed Issues (if relevant)

No issue 

### Manual testing scenarios (*)

Login to customer account page in mobile
Navigation should show in top and collapsible on touch for visible links navigate to another page

### Questions or comments


Before results
-Mobile
![customer_nav_mobile](https://user-images.githubusercontent.com/1908873/94579132-ad46fc00-02a2-11eb-8b28-14ac0e98c7e7.png)

-Desktop
![customer_nav_desktop](https://user-images.githubusercontent.com/1908873/94579148-b0da8300-02a2-11eb-8d71-f1de4fee0da8.png)

After results
-Mobile
![customer_nav_mobile_update](https://user-images.githubusercontent.com/1908873/94580365-082d2300-02a4-11eb-99ca-8c07b17df4e9.png)

-Desktop
Remain like before change

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#30237: Add collapsible nav in customer account theme blank
